addons/binding/org.openhab.binding.meterreader/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,43 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: MeterReader Binding
+Bundle-SymbolicName: org.openhab.binding.meterreader;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/jsml.jar,
+ lib/j62056.jar
+Import-Package: 
+ gnu.io,
+ javax.measure;version="1.0.0",
+ javax.measure.format;version="1.0.0",
+ javax.measure.quantity;version="1.0.0",
+ javax.measure.spi;version="1.0.0",
+ org.apache.commons.codec;version="1.6.0",
+ org.apache.commons.codec.binary;version="1.6.0",
+ org.apache.commons.lang;version="2.6.0",
+ org.apache.commons.net;version="3.2.0",
+ org.apache.commons.net.telnet;version="3.2.0",
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.common,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.library.unit,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.hamcrest;version="1.1.0";resolution:=optional,
```

Comment:
you must not depend on hamcrest and junit

Reply:
Done!